test/upload.js

```diff
@@ -1091,4 +1091,35 @@ describe("File upload", function () {
         expect(callback.calledOnce).to.be.true;
         sinon.assert.calledWith(callback, null, uploadSuccessResponseObj);
     });
+
+    it('With pre and post transformation', async function () {
+        const fileOptions = {
+            ...securityParameters,
+            fileName: "test_file_name",
+            file: "test_file",
+            responseFields: "tags, customCoordinates, isPrivateFile, metadata",
+            useUniqueFileName: false,
+            transformation: { pre: 'w-100', post: [{type: 'transformation', value: ''}]}
```

Comment:
It should result in error. why allow empty `value`?

Reply:
added checks for this and also added test suits to assert this